kythe/typescript/test.ts

```diff
@@ -175,28 +181,30 @@ function filterTestFileGroups(
 async function testIndexer(filters: string[], plugins?: indexer.Plugin[]) {
   const config =
       indexer.loadTsConfig('testdata/tsconfig.for.tests.json', 'testdata');
-  let testFilesGroups = getTestFileGroups();
+  let testFilesGroups = getTestFileGroups('testdata');
   if (filters.length !== 0) {
     testFilesGroups = filterTestFileGroups(testFilesGroups, filters);
   }
 
   const host = createTestCompilerHost(config.options);
   for (const testFiles of testFilesGroups) {
-    const testName = path.relative(config.options.rootDir!, testFiles[0]);
-    if (testName.endsWith('plugin.ts')) {
-      // plugin.ts is tested by testPlugin() test.
-      continue;
-    }
-    const start = new Date().valueOf();
-    process.stdout.write(`${testName}: `);
-    try {
-      await verify(host, config.options, testFiles, plugins);
-    } catch (e) {
-      console.log('FAIL');
-      throw e;
+    for (const testFile of testFiles) {
+      const testName = path.relative(config.options.rootDir!, testFile);
+      if (testName.endsWith('plugin.ts')) {
+        // plugin.ts is tested by testPlugin() test.
+        continue;
+      }
+      const start = new Date().valueOf();
+      process.stdout.write(`${testName}: `);
+      try {
+        await verify(host, config.options, testFiles, plugins);
+      } catch (e) {
+        console.log('FAIL');
+        throw e;
+      }
+      const time = new Date().valueOf() - start;
+      console.log('PASS', time + 'ms');
```

Comment:
JFYI, this generates a warning during import that would be nice to clean up at some point.
"Operands of '+' operation must either be both strings or both numbers or both bigints, but found number + "ms". Consider using template literals."

Reply:
thanks for the pointer, see #4593